client/webserver/site/src/js/wallets.ts

```diff
@@ -1316,3 +1330,20 @@ function assetIsConfigurable (assetID: number) {
   const zerothOpts = defs[0].configopts
   return defs.length > 1 || (zerothOpts && zerothOpts.length > 0)
 }
+
+/*
+ * customWalletBalance returns the translated string and a message for the
+ * provided balance category.
+ */
+function customWalletBalanceCategory (category: string): [string, string] {
+  switch (category) {
+    case 'Reserves Deficit':
+      return [intl.prep(intl.ID_RESERVES_DEFICIT), intl.prep(intl.ID_RESERVES_DEFICIT_MSG)]
+    case 'Bond Reserves':
+      return [intl.prep(intl.ID_BOND_RESERVES), intl.prep(intl.ID_BOND_RESERVES_MSG)]
+    case 'Shielded':
```

Comment:
If we are going to match on these strings, they should be made canonical in client/asset/interface.go with docs there so the coder knows that they are enumerated in one place since they may be recognized on the frontend and if they are ever changed, the consumer code needs to change too.